config/menus/glue.cfg

```diff
@@ -169,38 +169,44 @@ ui_two = [
 
 ui_big_macro = [ ui_stay_open [
```

Comment:
`ui_big_macro` needs documented, especially with the changes. What is happening here and inside?

Reply:
;_; you want me to ... understand this mess?

Reply:
Well, at least document what you changed.

Reply:
sigh, alright here goes my weekend xD